entrypoint.sh

```diff
@@ -118,6 +118,13 @@ INPUT_IDE_VERSIONS="$3"
 # failure-levels: ['COMPATIBILITY_PROBLEMS', 'INVALID_PLUGIN']
 FAILURE_LEVELS="$4"
 
+# Found from: https://github.com/JetBrains/intellij-plugin-verifier?tab=readme-ov-file#specific-options
+#
+# Input string can be a comma separated list
+#
+# mute-plugin-problems: ['ForbiddenPluginIdPrefix', 'TemplateWordInPluginId', 'TemplateWordInPluginName']
```

Comment:
Is this actually how the yaml of the action in a workflow file would look like?

I assumed it would instead look something like below:
```suggestion
# mute-plugin-problems: 'ForbiddenPluginIdPrefix,TemplateWordInPluginId,TemplateWordInPluginName'
```

Is that not the case?

Reply:
Ah, yes! My mistake, that is correct.